### ISSUE TITLE
Player is Not Defined

### ISSUE THREAD

**Issue by jah2488** [ippa/jaws#4]
Hey ippa,
This jaws library is amazing, but i've run into a bug that I simply cannot figure out and thought I'd get your take on it.

The game keeps pausing at `100% loaded` and when I check my console it tells me, `game.js:28Uncaught ReferenceError: player is not defined`

Its very odd, because my player is defined and I cannot for the life of me figure this out.

Thanks

My `game.js` file

``` js
function Game() {
  this.setup = function() {
    this.player = new jaws.Sprite({
      image: "plane.png",
      x: jaws.width / 2,
      y: jaws.height / 2,
      anchor: "center"
    });
    jaws.preventDefaultKeys(["up", "down", "left", "right"]);
    return;
  };
  this.update = function() {
    if (jaws.pressed("left")) {
      player.x -= 2;
    }
    if (jaws.pressed("right")) {
      player.x += 2;
    }
    if (jaws.pressed("up")) {
      player.y -= 2;
    }
    if (jaws.pressed("down")) {
      player.y += 2;
    }
    return;
  };
  this.draw = function() {
    player.draw(); // This is the line calling the error ------------------------
    return;
  };
  return;
};
window.onload = function() {
  jaws.assets.add("plane.png");
  jaws.start(Game);
  return;
};
```

and my index.html file

``` html
<html>
  <head>
    <title>Test Game with Jaws Framework</title>
  <script src="jaws.js" type="text/javascript"></script> 
  </head>
  <body>
    <canvas></canvas>
    <script src="game.js"></script> 
  </body>
</html>
```
